modules/codelite/codelite_project.lua

```diff
@@ -117,10 +117,16 @@
 		tree.traverse(tr, {
 			-- folders are handled at the internal nodes
 			onbranchenter = function(node, depth)
-				_p(depth, '<VirtualDirectory Name="%s">', node.name)
+				if(depth > 0)
+				then
```

Comment:
This is inconsistent with the rest of the code base. `if depth > 0 then` would be consistent with the rest of the code base.